.travis.yml

```diff
@@ -1,8 +1,5 @@
 language: node_js
-sudo: required
-before_script:
-  - npm install -g polymer-cli
-  - polymer install --variants
+sudo: false
```

Comment:
Why is this also changing as part of this PR? We prefer keeping all the `travis.yml` files consistent across all elements, so if one needs to change, they all should, via a `tedium` pass. In any case, it doesn't belong in this PR